src/lib/statistics/statistics_objects/min_max_filter.cpp

```diff
@@ -160,6 +160,9 @@ bool MinMaxFilter<T>::does_not_contain(const PredicateCondition predicate_condit
       return value >= max || value2 <= min;
     }
     case PredicateCondition::Like: {  // NOLINTNEXTLINE - clang-tidy doesn't like the else path of the if constexpr
+      // We do not handle the case of outlier min/max values. For example `USA%` is not pruned whenever just a single
```

Comment:
I think the outlier term is confusing. min/max are per se "outliers".

Maybe something like "We use the ascii collation for min/max filters. This means that lower case letters are considered larger than upper case letters. This can lead to a situation, where the `USA%` (e.g., JOB query XXXXX) is not pruned whenever just a single value starts with a lower case letter."